src/compute.geometry/IO/Schema.cs

```diff
@@ -8,6 +8,12 @@ public class Schema
     {
         public Schema() {}
 
+        [JsonProperty(PropertyName = "absolutetolerance")]
```

Comment:
These should both have a default initial value of 0. I know .NET does initialize properties to 0, but I would rather have it explicit in the code
`public double AbsoluteTolerance { get; set; } = 0;`
This way clients other than Hops that don't know about these tolerances can choose not to use them.